tests/TestCase/ORM/TableTest.php

```diff
@@ -6012,6 +6012,18 @@ public function testLoadIntoMany()
     }
 
     /**
+     * Test getting the save options builder.
+     *
+     * @return void
+     */
+    public function getSaveOptionsBuilder()
+    {
+        $table = TableRegistry::get('Authors');
+        $result = $table->getSaveOptionsBuilder();
+        $this->assertInstanceOf('Cake\ORM\SaveOptionsBuilder', $result);
```

Comment:
Could we have an integration test for the saveOptionBuilder() being used with save()?